src/WCSimDetectorConfigs.cc

```diff
@@ -219,8 +219,11 @@ void WCSimDetectorConstruction::SetHyperKGeometry()
   WCPMTName           = PMT->GetPMTName();
   WCPMTExposeHeight   = PMT->GetExposeHeight();
   WCPMTRadius         = PMT->GetRadius();
-  WCIDDiameter          = 70.8*m; // = 74m - 2*(60cm ID wall + 1m OD)
-  WCIDHeight            = 54.8*m; // = 60m - 2*(60cm ID wall + 2m OD)
+  //WCIDDiameter          = 70.8*m; // = 74m - 2*(60cm ID wall + 1m OD)
+  //WCIDHeight            = 54.8*m; // = 60m - 2*(60cm ID wall + 2m OD)
```

Comment:
Put back the default value. (yes, we need to still put in the new dimensions, but this will be separate)

Reply:
Done